wgsl/index.bs

```diff
@@ -1058,52 +1053,95 @@ case_body
 
 <pre class='def'>
 loop_stmt
-  : LOOP BRACKET_LEFT statements continuing_stmt? BRACKET_RIGHT
-
-continuing_stmt:
-  : CONTINUING body_stmt
+  : LOOP BRACKET_LEFT statements BRACKET_RIGHT
 </pre>
 
 The loop statement is one of the biggest divergences from other shader
 languages. The structure of the statement is designed to closely match the block
 ordering requirements for a loop in SPIR-V.
 
+
+<div class='example' heading='General GLSL Loop Form'>
+  <xmp>
+    for (A; B; C) {
+      D
+      if (E) break;
+      if (F) continue;
+      G
+      break;
+    }
+  </xmp>
+</div>
+
+<div class='example' heading="General [SHORTNAME] Loop Form">
+  <xmp>
+    A
```

Comment:
Wrap this whole thing in `{}` to contain the scope of `A`?

Reply:
Ah, yeah, that would be more precise.